pallets/sequencer-staking/src/lib.rs

```diff
@@ -492,20 +492,11 @@ impl<T: Config> Pallet<T> {
 	// 	Self::maybe_remove_sequencers_from_active_set(exiting_collators);
 	// }
 
-	fn maybe_remove_sequencers_from_active_set(exiting_collators: BTreeSet<T::AccountId>) {
-		let to_remove = ActiveSequencers::<T>::get().into_iter().map(|(chain, active)| {
-			(
-				chain,
-				active
-					.iter()
-					.filter(|seq| exiting_collators.contains(seq))
-					.cloned()
-					.collect::<BTreeSet<_>>(),
-			)
-		});
-
-		for (chain, sequencers_set) in to_remove {
-			Self::remove_sequencers_from_active_set(chain, sequencers_set);
+	fn maybe_remove_sequencer_from_active_set(chain: T::ChainId, sequencer: T::AccountId) {
```

Comment:
Sure, yeah you can change this a single sequencer function. This was mostly for eligibility where multiple guys could leave in a single instance...